### PR TITLE
Setup Docker and CircleCI

### PR DESCRIPTION
**Pull request by christophermanahan**
- Setup Docker and application configurations for development and production environments
- Setup CircleCI to run unit tests as part of a continuous integration pipeline
- Deployed to http://ebae.us-east-2.elasticbeanstalk.com/

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+version: 2  # use CircleCI 2.0 instead of CircleCI Classic
+jobs:  # basic units of work in a run
+  build:  # runs not using Workflows must have a `build` job as entry point
+    parallelism: 1  # run only one instance of this job in parallel
+    docker:  # run the steps with Docker
+      - image: circleci/elixir:latest  # ...with this image as the primary container; this is where all `steps` will run
```
Comment (ssunday) [inline .circleci/config.yml]: I would say to tag this with specific version

Comment (ssunday) [reply .circleci/config.yml]: your code may not always work with latest

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# ./Dockerfile
+
+FROM elixir:latest
```
Comment (ssunday) [inline Dockerfile]: same deal

File: .circleci/config.yml (hunk 1 above)
Comment (christophermanahan) [reply .circleci/config.yml]: Makes sense, thanks!

**Pull request merged**